javascript-modal/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Javascript Light Bulb</title>
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+    <div class="row">
+      <input type="button" value="Open Modal" class="modal-button font-larger">
```

Comment:
We're not working with `form`s here, so this should just be a normal `button`. You should only be using `input` when working with a `form` or collecting data